packages/flutter/test/widgets/list_wheel_scroll_view_test.dart

```diff
@@ -238,6 +253,30 @@ void main() {
   });
 
   group('viewport transformation', () {
+    testWidgets('Center child is magnified', (WidgetTester tester) async {
```

Comment:
https://github.com/flutter/flutter/issues/17483
You might need to skip the test when not running on Linux

Reply:
fixed.